spec/features/admin_views_users_spec.rb

```diff
@@ -5,22 +5,23 @@
   I want to view a list of users
   So that I can keep track of the site
 } do
-  
-  scenario 'admin views all users' do
-    admin = FactoryGirl.create(:admin_user)
-    
-    user = FactoryGirl.create(:user)
-    user1 = FactoryGirl.create(:user)
 
-    sign_in_as(admin)
+  let!(:admin) { FactoryGirl.create(:admin_user) }
+  let!(:user) { FactoryGirl.create(:user) }
+  let!(:user1) { FactoryGirl.create(:user) }
 
+  scenario 'admin views all users' do
+    sign_in_as(admin)
     visit admin_users_path
-   
-    expect(page).to have_content user.email
-    expect(page).to have_content user1.email
-    
+    expect(page).to have_content user.name
+    expect(page).to have_content user1.name
   end
 
-  
+  scenario 'authenticated user tries to view all users' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.